baselines/teams.md

```diff
@@ -339,40 +385,34 @@ Blog](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/skype-for-busi
 
 ### Implementation
 
-Instructions for *enabling* communications with Skype users are outlined
-in [Communicate with Skype
-users](https://docs.microsoft.com/en-us/microsoftteams/manage-external-access#communicate-with-skype-users).
+All the settings in this section are configured in the [**Microsoft Teams admin
+    center**](https://admin.teams.microsoft.com).
+
+#### MS.TEAMS.6.1v1 instructions:
 
 1.  Sign in to the **[Microsoft Teams admin
     center](https://admin.teams.microsoft.com).**
 
-2.  Select **Users -\> External access.**
+2.  Select **Users > External access.**
 
 3.  Under **Skype** users, set **Allow users in my organization to
     communicate with Skype users** to **Off**.
 
 4.  Click **Save**.
 
 ## 7. Teams Email Integration
-
-Teams provides an optional feature that allows channels to have an email
-address and receive email. These channel email addresses are not under
-the tenant’s domain; rather, they are associated with a Microsoft-owned
-domain, teams.ms. As such, although some basic checks are performed,
-agencies do not have control over the security settings associated with
-this email. For this reason, email channel integration should be
-disabled.
+This section helps reduce security risks related to teams email integration. Teams provides an optional feature that allows channels to have an email address and receive email.
 
 ### Policies
 #### MS.TEAMS.7.1v1
 Teams email integration SHALL be disabled.
-- _Rationale:_ TODO
+- _Rationale:_ The security risk with Microsoft Team's email integration is due to the fact that Teams channel's email addresses are not under the tenant’s domain, rather they are associated with the Microsoft-owned domain of "teams.ms". By disabling Teams email integration the risk of use of emails outside of the agency's security settings control is minimized.
```

Comment:
I do not understand what is meant by "By disabling Teams email integration the risk of use of emails outside of the agency's security settings control is minimized."
Needs clarity.